### DISCUSSION TITLE
"Return" to send command to gemini, "Shift + Return" to create new line

### DISCUSSION THREAD

**Discussion by antkss** [end-4/dots-hyprland#324] [General]
![20240312_11h37m12s_grim](https://github.com/end-4/dots-hyprland/assets/88892713/3b6a00ae-d74a-49df-b07d-20b36770b555)

**Comment by end-4** [comment]
yeah that's intended behavior

**Reply by antkss** [reply]
how about making the behavior of this become the behavior of gemini text box??

![image](https://github.com/end-4/dots-hyprland/assets/88892713/2d33ab04-a1a4-4dbb-abe8-648ae572814a)

**Reply by end-4** [reply]
well
another item in my never-ending todo list

**Comment by antkss** [comment]
[

https://github.com/end-4/dots-hyprland/assets/88892713/3f2a028d-0c99-45a4-95e7-c53081e92e29

](url)
i don't know how to code javascript, but if there something wrong about this, please let's me know, because i just use the overview textbox to replace your old text box, why didn't you do this ?

**Reply by end-4** [reply]
what are you trying to do anyway

**Reply by daUnknownCoder** [reply]
yeah what was that `turn me on` about

**Reply by antkss** [reply]
because i think it's faster when typing without click =)), if i don't use gemini to be faster, there's no reason i chose this dotfiles =))), it made me feel a lot more comfortable when i solved my problem, i learnt something

**Comment by daUnknownCoder** [comment]
and yeah [OP](@antkss), can i get thy wallpaper?

**Reply by antkss** [reply]
yes of course 
![ori](https://github.com/end-4/dots-hyprland/assets/88892713/7d5f8bef-9883-491f-a3e3-62420d35d588)
